package.json

```diff
@@ -87,6 +87,6 @@
     "node": ">=18.0.0"
   },
   "prisma": {
-    "seed": "ts-node --require tsconfig-paths/register prisma/seed.ts"
+    "seed": "ts-node -r tsconfig-paths/register prisma/seed.ts"
```

Comment:
both `ts-node` and `bun` support `-r`, but of the two only `ts-node` supports `--require` (the long name in `bun` is `--preload`).